pkg/model/apikey.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 const (
-	apiKeyLength = 50
+	apiKeyLength = 32
```

Comment:
does this change due to some change in requirement?

Reply:
To be honest, I'm wondering about the APIKey's length.
Currently, the returned APIKey contains 2 parts like this "UUID-PART.RANDOM-PART".
- The first part is a UUID string used as the key's ID with a length of 32 characters.
- The second part is a random string used as the key's password.

If the random part is 50 characters then the whole APIKey looks pretty long.
So I reduced the length of that part to 32 and the entire length of the APIKey is 64 characters.

(Not sure, but maybe we also should encrypt the APIKey (e.g. base64) to get a better look 🤔 )

Reply:
> (Not sure, but maybe we also should encrypt the APIKey (e.g. base64) to get a better look 🤔 )
I think it would be better if the key has a specific length in all case, will reduce other issues around miss/conflict setting. What do you think? 🤔

Reply:
You are right. Nice catch!

Reply:
let's address it by another PR 😄